lib/surface/compiler/eex_engine.ex

```diff
@@ -341,7 +341,7 @@ defmodule Surface.Compiler.EExEngine do
       for {name, size, infos} <- slot_info do
         prop_assign_mappings =
           Enum.map(infos, fn {let, _, _} ->
-            Enum.map(let, fn {prop_name, {binding_name, _, _}} -> {prop_name, binding_name} end)
+            Enum.map(let, fn {prop_name, binding_name} -> {prop_name, binding_name} end)
```

Comment:
if this is already a keyword list of prop -> binding then we don't need this inner map anymore

Reply:
🤦‍♂️